tests/unit_test/channels/whatsapp_test.py

```
@@ -0,0 +1,270 @@
+                assert args[0] == bot
+                user_message = args[1]
+
+                assert user_message.text == '/k_multimedia_msg{"document": "sdfghj567"}'
+
+    @pytest.mark.asyncio
+    async def test_payment_message_request(self):
+        from kairon.chat.handlers.channels.whatsapp import Whatsapp, WhatsappBot
+        from kairon.shared.chat.data_objects import ChannelLogs
+
+        with patch.object(WhatsappBot, "mark_as_read"):
+            with patch.object(Whatsapp, "process_message") as mock_message:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)